set-definitions/src/main/java/eu/europeana/set/definitions/model/vocabulary/UserSetTypes.java

```diff
@@ -10,7 +10,8 @@
  */
 public enum UserSetTypes implements JsonKeyword {
 
-    COLLECTION("Collection"), DYNAMICCOLLECTION("DynamicCollection"), BOOKMARKSFOLDER("BookmarkFolder"), ENTITYBESTITEMSSET("EntityBestItemsSet");
+    COLLECTION("Collection"), DYNAMICCOLLECTION("DynamicCollection"), BOOKMARKSFOLDER("BookmarkFolder"), ENTITYBESTITEMSSET("EntityBestItemsSet"), 
+    GALLERY("Gallery");
```

Comment:
Gallery must not be included here. Only the values used in the type must be included here. For the time being we can use a constant in WebUserSetModelFields